client/my-sites/stats/stats-subscribers-chart-section/index.tsx

```diff
@@ -1,9 +1,9 @@
 import config from '@automattic/calypso-config';
-import UplotChart from '@automattic/components/src/chart-uplot';
```

Comment:
`@automattic/components/src` 😱 

I just can't wrap my head around how this line was allowed in the first place.